servers/rendering/renderer_rd/environment/gi.cpp

```diff
@@ -1866,6 +1866,10 @@ void GI::SDFGI::pre_process_gi(const Transform3D &p_transform, RenderDataRD *p_r
 
 	RD::get_singleton()->buffer_update(gi->sdfgi_ubo, 0, sizeof(SDFGIData), &sdfgi_data);
 
+	if (p_render_data->sdfgi_update_data == nullptr) {
+		return;
+	}
```

Comment:
I would do this check at the top of the function. We can't use SDFGI when `sdfgi_update_data` is a nullptr anyway, so we might as well skip the whole function